### ISSUE TITLE
Txt format Google Drive file is truncated and incomplete

### ISSUE THREAD

**Issue by yieldthought** [CorentinJ/librispeech-alignments#5]
At least one of the files in 68MB file at https://drive.google.com/file/d/1WYfgr31T-PPwMcxuAq09XZfHQO5Mw8fE/view?usp=sharing is truncated in the middle of the list of intervals with several lines missing and no closing "

**Comment by Emrys365** [comment]
I also observe that this file `LibriSpeech/train-other-500/551/129024/1618-129024.alignment.txt` in the downloaded zip is incomplete:
> The last line is not enclosed with `"`.
```
551-129024-0000 ",CHAPTER,SEVENTEEN,,HOW,CAPTAIN,DOBBIN,,BOUGHT,A,PIANO,,IF,THERE,IS,,ANY,EXHIBITION,IN,,ALL,VANITY,FAIR,WHICH,SATIRE,AND,SENTIMENT,CAN,VISIT,ARM,IN,ARM,TOGETHER,,WHERE,YOU,LIGHT,ON,THE,STRANGEST,CONTRASTS," "0.160,0.680,1.430,2.090,2.310,2.850,3.470,3.520,3.800,3.840,4.490,6.130,6.310,6.480,6.600,6.670,6.960,7.570,7.750,7.790,8.060,8.500,8.840,9.110,9.670,9.820,10.300,10.450,10.840,11.140,11.240,11.530,12.100,12.310,12.640,12.820,13.250,13.410,13.520,14.270,14.910,15.005" 
551-129024-0001 ",LAUGHABLE,AND,TEARFUL,,WHERE,YOU,MAY,BE,GENTLE,AND,PATHETIC,,OR,SAVAGE,AND,CYNICAL,,WITH,PERFECT,PROPRIETY,,IT,IS,AT,ONE,OF,THOSE,PUBLIC,ASSEMBLIES,,A,CROWD,OF,WHICH,ARE,ADVERTISED,EVERY,DAY,IN,THE,LAST,PAGE,OF,THE,TIMES,NEWSPAPER," "0.070,0.790,1.020,1.600,1.870,2.150,2.320,2.500,2.660,3.150,3.240,3.860,4.070,4.280,4.730,4.890,5.500,5.800,6.050,6.450,7.320,7.980,8.150,8.310,8.460,8.640,8.710,8.930,9.360,10.160,10.470,10.590,10.950,11.030,11.230,11.340,12.030,12.320,12.580,12.660,12.750,13.080,13.400,13.490,13.590,14.000,14.670,14.89" 
551-129024-0002 ",AND,,OVER,WHICH,THE,LATE,MISTER,GEORGE,ROBINS,USED,TO,PRESIDE,WITH,SO,MUCH,DIGNITY,,THERE,ARE,VERY,FEW,LONDON,PEOPLE,,AS,I,FANCY,,WHO,HAVE,NOT,ATTENDED,AT,THESE,MEETINGS,,AND,,ALL,WITH,A,TASTE,FOR,MORALIZING,MUST,HAVE,THOUGHT," "0.130,0.350,0.380,0.630,0.830,0.940,1.230,1.520,1.870,2.400,2.560,2.640,3.200,3.320,3.560,3.790,4.420,5.550,5.800,5.890,6.300,6.540,6.970,7.520,7.580,7.790,7.970,8.510,8.850,8.940,9.110,9.420,9.960,10.140,10.440,11.060,11.420,11.680,11.750,12.060,12.230,12.290,12.610,12.730,13.570,13.830,14.000,14.420,14.595" 
551-129024-0003 ",WITH,A,SENSATION,AND,INTEREST,NOT,A,LITTLE,STARTLING,AND,QUEER,,OF,THE,DAY,WHEN,THEIR,TURN,SHALL,COME,TOO," "0.160,0.350,0.430,1.070,1.190,1.640,1.900,2.020,2.290,2.860,3.000,3.500,3.730,4.020,4.130,4.380,4.640,4.940,5.330,5.550,5.860,6.270,6.37" 
551-129024-0004 ",OR,WILL,BE,INSTRUCTED,BY,THE,EXECUTORS,,TO,OFFER,TO,PUBLIC,COMPETITION,,THE,LIBRARY,,FURNITURE,,PLATE,,WARDROBE," "0.170,0.290,0.530,0.690,1.310,1.480,1.610,2.420,2.680,2.890,3.320,3.510,3.940,4.670,4.870,5.010,5.650,5.940,6.620,6.780,7.270,7.670,8.390,8.485" 
551-129024-0005 ",EVEN,WITH,THE,MOST,SELFISH,DISPOSITION,,THE,VANITY,FAIRIAN," "0.240,0.630,0.800,0.890,1.220,1.650,2.440,2.680,2.790,3.240,3.830,3.945" 
551-129024-0006 ",MY,LORD,DIVES'S,REMAINS,ARE,IN,THE,FAMILY,VAULT,,THE,STATUARIES,ARE,CUTTING,AN,INSCRIPTION,VERACIOUSLY,COMMEMORATING,HIS,VIRTUES,,AND,THE,SORROWS,OF,HIS,HEIR,,WHO,IS,DISPOSING,OF,HIS,GOODS," "0.310,0.560,0.940,1.490,1.960,2.060,2.140,2.210,2.660,3.130,3.460,3.600,4.430,4.520,4.920,5.030,5.810,6.500,7.290,7.510,8.140,8.410,8.620,8.740,9.290,9.380,9.600,9.950,10.090,10.260,10.330,11.030,11.130,11.300,11.820,12.25" 
551-129024-0007 ",THE,FAMILIAR,HOUSE,OF,WHICH,THE,LIGHTS,USED,TO,SHINE,SO,CHEERFULLY,AT,SEVEN,O'CLOCK,,OF,WHICH,THE,HALL,DOORS,OPENED,SO,READILY," "0.330,0.430,1.020,1.420,1.530,1.720,1.920,2.330,2.500,2.600,2.990,3.300,3.770,3.880,4.180,4.730,5.240,5.410,5.590,5.730,6.040,6.410,6.780,6.920,7.590,7.725" 
551-129024-0008 ",AS,YOU,PASSED,UP,THE,COMFORTABLE,STAIR,,SOUNDED,YOUR,NAME,FROM,LANDING,TO,LANDING,,UNTIL,IT,REACHED,THE,APARTMENT,WHERE,JOLLY,OLD,DIVES,WELCOMED,HIS,FRIENDS,,WHAT,A,NUMBER,OF,THEM,HE,HAD,,AND,WHAT,A,NOBLE,WAY,,OF,ENTERTAINING,THEM," "0.120,0.330,0.440,0.810,0.930,1.030,1.500,2.020,2.280,2.730,2.920,3.220,3.450,3.830,3.990,4.450,4.770,5.100,5.180,5.470,5.600,6.060,6.220,6.690,6.990,7.440,7.940,8.120,8.730,9.300,9.530,9.580,10.060,10.190,10.380,10.530,10.930,11.260,11.420,11.610,11.670,12.350,12.790,12.820,13.010,13.670,13.970,14.3" 
551-129024-0009 ",AND,HOW,COURTEOUS,AND,FRIENDLY,MEN,WHO,SLANDERED,AND,HATED,EACH,OTHER,EVERYWHERE,ELSE," "0.280,0.460,0.650,1.260,1.470,1.830,2.070,2.240,2.890,3.010,3.420,3.660,3.900,4.380,4.840,5.135" 
551-129024-0010 ",HE,WAS,RATHER,DULL,PERHAPS,,BUT,WOULD,NOT,SUCH,WINE,MAKE,,ANY,CONVERSATION,PLEASANT,,WE,MUST,,GET,SOME,OF,HIS,BURGUNDY,AT,,ANY,PRICE,,THE,MOURNERS,CRY,AT,HIS,CLUB," "0.300,0.470,0.740,1.080,1.480,2.150,2.260,2.430,2.680,3.030,3.320,3.790,4.060,4.090,4.380,5.090,5.670,6.360,6.510,7.030,7.090,7.320,7.520,7.620,7.800,8.330,8.470,8.500,8.760,9.170,9.200,9.270,9.740,10.020,10.090,10.240,10.680,11.18" 
551-129024-0011 ",PINCHER,SAYS,HANDING,IT,ROUND," "0.160,0.550,0.980,1.460,1.560,2.040,2.25" 
551-129024-0012 ",PRETTY,THING,IS,IT,NOT,,SWEET,MINIATURE," "0.130,0.390,0.750,0.890,0.960,1.400,1.680,2.120,2.860,3.15" 
551-129024-0013 ",HOW,CHANGED,THE,HOUSE,IS,THOUGH,,THE,FRONT,IS,PATCHED,OVER,WITH,BILLS," "0.470,0.770,1.350,1.440,1.780,2.030,2.380,2.730,2.850,3.180,3.320,3.730,3.980,4.190,4.750,4.845" 
551-129024-0014 ",THE,HALL,SWARMS,WITH,DINGY,GUESTS,OF,,ORIENTAL,COUNTENANCE," "0.130,0.230,0.550,1.240,1.430,1.780,2.140,2.280,2.350,2.910,3.560,3.715" 
551-129024-0015 ",OLD,WOMEN,AND,AMATEURS,HAVE,INVADED,THE,UPPER,APARTMENTS,,PINCHING,THE,BED,CURTAINS,,POKING,INTO,THE,FEATHERS,,SHAMPOOING,THE,MATTRESSES,,AND,CLAPPING,THE,WARDROBE,,DRAWERS,TO,AND,FRO," "0.520,0.840,1.140,1.290,2.000,2.230,2.890,3.010,3.250,3.970,4.530,4.960,5.070,5.410,5.960,6.270,6.750,6.960,7.060,7.640,7.900,8.670,8.750,9.550,9.690,9.850,10.270,10.340,10.990,11.040,11.670,12.070,12.240,12.730,13.025" 
551-129024-0016 ",ENTERPRISING,YOUNG,HOUSEKEEPERS,ARE,MEASURING,THE,LOOKING,GLASSES,AND,HANGINGS," "0.350,1.030,1.250,1.960,2.050,2.560,2.680,3.050,3.520,3.650,4.280,4.4" 
551-129024-0017 ",AND,MISTER,HAMMERDOWN,IS,SITTING,ON,THE,GREAT,MAHOGANY,DINING,TABLES,,IN,THE,DINING,ROOM,BELOW,,WAVING,THE,,IVORY,HAMMER,,AND,,EMPLOYING,ALL,THE,ARTIFICES,OF,,ELOQUENCE,,ENTHUSIASM,,ENTREATY,,REASON,,DESPAIR," "0.220,0.460,0.830,1.600,1.740,2.050,2.190,2.320,2.620,3.190,3.590,4.120,4.390,4.560,4.700,5.100,5.290,5.810,6.090,6.540,6.680,6.730,7.220,7.680,7.930,8.170,8.200,8.770,9.040,9.190,9.750,9.880,9.940,10.540,10.750,11.750,11.930,12.620,12.840,13.440,13.670,14.460,14.475" 
551-129024-0018 ",SHOUTING,TO,HIS,PEOPLE,,SATIRIZING,MISTER," "0.120,0.650,0.770,0.910,1.430,1.670,2.570,2.880,2.93" 
551-129024-0019 ",INSPIRITING,MISTER,MOSS,INTO,ACTION,,IMPLORING,,COMMANDING,,BELLOWING," "0.220,1.110,1.410,1.940,2.190,2.690,3.040,3.740,3.990,4.720,5.090,5.670,5.81" 
551-129024-0020 ",AND,WE,PASS,TO,THE,NEXT,LOT,,O,DIVES,,WHO,WOULD,,EVER,HAVE,THOUGHT,,AS,WE,SAT,ROUND,THE,BROAD,TABLE,SPARKLING,WITH,PLATE,AND,SPOTLESS,LINEN," "0.120,0.240,0.340,0.770,0.900,0.990,1.340,1.720,2.580,2.970,3.630,3.880,4.060,4.290,4.430,4.690,4.960,5.410,5.630,5.890,6.020,6.350,6.610,6.680,7.100,7.460,8.040,8.270,8.610,8.760,9.350,9.750,9.945" 
551-129024-0021 ",THE,,EXCELLENT,DRAWING,ROOM,FURNITURE,BY,THE,BEST,MAKERS,,THE,RARE,AND,FAMOUS,WINES,SELECTED,REGARDLESS,OF,COST,,AND,WITH,THE,WELL,KNOWN,TASTE,OF,THE,PURCHASER,,THE,RICH,AND,COMPLETE,SET,OF,,FAMILY,PLATE,HAD,BEEN,SOLD,ON,THE,PREVIOUS,DAYS," "0.300,0.470,0.500,0.940,1.360,1.600,2.080,2.250,2.360,2.760,3.350,3.630,3.790,4.190,4.330,4.720,5.140,5.740,6.310,6.420,6.980,7.310,7.440,7.550,7.650,7.950,8.230,8.650,8.720,8.780,9.490,9.950,10.070,10.360,10.490,10.940,11.250,11.330,11.380,11.790,12.160,12.250,12.440,12.880,13.010,13.080,13.570,14.180,14.695" 
551-129024-0022 ",CERTAIN,OF,THE,BEST,WINES,,WHICH,ALL,HAD,A,GREAT,CHARACTER,AMONG,AMATEURS,IN,THE,NEIGHBOURHOOD,,HAD,BEEN,PURCHASED,FOR,HIS,MASTER,,WHO,KNEW,THEM,VERY,WELL,,BY,THE,BUTLER,OF,OUR,FRIEND,JOHN,OSBORNE,,ESQUIRE,,OF,RUSSELL,SQUARE," "0.510,0.920,1.030,1.110,1.450,2.010,2.250,2.450,2.660,2.830,2.910,3.210,3.590,3.870,4.540,4.640,4.760,5.440,5.870,6.080,6.260,6.760,6.900,7.040,7.600,7.630,7.760,8.130,8.340,8.640,9.180,9.210,9.360,9.480,10.030,10.150,10.290,10.790,11.200,11.920,12.150,12.900,13.140,13.330,13.640,14.200,14.56" 
551-129024-0023 ",IT,HAPPENED,THAT,THE,ORATOR,ON,THE,TABLE,WAS,EXPATIATING,ON,THE,MERITS,OF,A,PICTURE,,WHICH,HE,SOUGHT,TO,RECOMMEND,TO,HIS,AUDIENCE,,IT,WAS,BY,NO,MEANS,SO,SELECT,,OR,NUMEROUS,A,COMPANY,AS,HAD,ATTENDED,THE,PREVIOUS,DAYS,OF,THE,AUCTION," "0.210,0.340,0.800,0.950,1.050,1.590,1.700,1.780,2.070,2.250,3.090,3.190,3.260,3.670,3.750,3.850,4.310,4.530,4.880,4.990,5.300,5.410,5.980,6.080,6.250,6.940,7.470,7.550,7.710,7.920,8.220,8.640,8.830,9.520,9.590,9.780,10.240,10.290,10.670,10.810,11.010,11.600,11.700
```

**Comment by btamm12** [comment]
something has gone very wrong in this directory
```
LibriSpeech/train-other-500/551/129024/
```

There should only be one file named `551-129024.alignment.txt` but there are 1545 files (identical file content, different speaker ID's in the file name).
